corehq/apps/user_importer/helpers.py

```diff
@@ -215,19 +225,28 @@ def _fmt_phone(phone_number):
 class WebUserImporter(BaseUserImporter):
     def add_to_domain(self, role_qualified_id, location_id):
         self.user.add_as_web_user(self.user_domain, role=role_qualified_id, location_id=location_id)
-        self.logger.add_change_message(_("Added as web user"))
+        self.role_updated = bool(role_qualified_id)
+
+        self.logger.add_change_message(_("Added as web user to {domain_name}").format(
```

Comment:
nit: quote domain name
```suggestion
        self.logger.add_change_message(_("Added as web user to '{domain_name}'").format(
```

Reply:
https://github.com/dimagi/commcare-hq/pull/29935/commits/15e180758f65e1ac9a952e36fc8de8bedf73fa46